db/class.go

```diff
@@ -421,3 +421,65 @@ func (d *DB) DeleteClass(c echo.Context) error {
 
 	return c.String(http.StatusOK, "")
 }
+
+// GetClassMembers returns the user IDs and display names of each member in the requested class.
+// It takes a class id and checks that the given uid is in the class first
+func (d *DB) GetClassMembers(c echo.Context) error {
+	var (
+		req struct {
+			CID string `json:"cid"`
+			UID string `json:"uid"`
+		}
+
+		res map[string]User
+		err error
+	)
```

Comment:
You can probably get by removing the extra variable declarations here then use the short assignment statement, as you do later in the function.
```suggestion
	var req struct {
		CID string `json:"cid"`
		UID string `json:"uid"`
	}
```